### PR TITLE
Adds a bone tumor specific to slime people

### PR DESCRIPTION
**Pull request by Social-Moth**
## What Does This PR Do

 Adds Crystalized Slime Jelly, a specific kind of bone tumor that slime people get when they overdose on osseous reagent

## Why It's Good For The Game
Slime People don't have bones, them getting bone tumors challenges everything we know about their biology.

## Testing
OD skrell, OD slime, both get the right organs, and the server doesn't crash

## Changelog
:cl:
add: Slime People now get crystalized slime jelly instead of bone tumors
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/medicine.dm [hunk 1]
```diff
@@ -1019,7 +1019,9 @@
 
 	if(ishuman(M) && prob(5))
 		var/mob/living/carbon/human/H = M
-		if(!H.get_int_organ(/obj/item/organ/internal/bone_tumor))
+		if(isslimeperson(H) && !H.get_int_organ(/obj/item/organ/internal/bone_tumor/slime_tumor))
+			new/obj/item/organ/internal/bone_tumor/slime_tumor(H)
+		else if	(!H.get_int_organ(/obj/item/organ/internal/bone_tumor))
```
Comment (DGamerL) [inline code/modules/reagents/chemistry/reagents/medicine.dm:1024]:
```suggestion
		if(!H.get_int_organ(/obj/item/organ/internal/bone_tumor))
			if(isslimeperson(H))
				new /obj/item/organ/internal/bone_tumor/slime_tumor(H)
			else
```

File: code/modules/reagents/chemistry/reagents/medicine.dm [hunk 2]
```diff
@@ -1019,7 +1019,9 @@
...
 			new/obj/item/organ/internal/bone_tumor(H)
```
Comment (DGamerL) [inline code/modules/reagents/chemistry/reagents/medicine.dm:1025]:
```suggestion
				new/obj/item/organ/internal/bone_tumor(H)
```

File: code/modules/surgery/organs/bone_tumor.dm [hunk 3]
```diff
@@ -27,3 +27,27 @@
 
 		for(var/obj/item/organ/internal/I in other_groin_organs)
 			I.receive_damage(rand(5, 15) + tumor_damage_modifier)
+/obj/item/organ/internal/bone_tumor/slime_tumor
```
Comment (DGamerL) [inline code/modules/surgery/organs/bone_tumor.dm:30]:
```suggestion

/obj/item/organ/internal/bone_tumor/slime_tumor
```

File: code/modules/surgery/organs/bone_tumor.dm [hunk 4]
```diff
@@ -27,3 +27,27 @@
...
+	name = "crystalized slime jelly"
+	parent_organ = "groin"
+	slot = "groin_tumor"
+	destroy_on_removal = TRUE //Changed in New to be dependent on amount of osseous reagent in the system when formed, determines damage done
```
Comment (DGamerL) [inline code/modules/surgery/organs/bone_tumor.dm:34]:
```suggestion
```

File: code/modules/surgery/organs/bone_tumor.dm [hunk 5]
```diff
@@ -27,3 +27,27 @@
...
+		to_chat(owner, "<span class='danger'>Something sharp is moving around in your lower body!</span>")
+
+	if(prob(1))
+		to_chat(owner, "<span class='userdanger'>Something just tore in your lower body!</span>")
+
+		var/list/other_groin_organs = G.internal_organs
+		other_groin_organs -= src
+
+		for(var/obj/item/organ/internal/I in other_groin_organs)
+			I.receive_damage(rand(5, 15) + tumor_damage_modifier)
```
Comment (DGamerL) [inline code/modules/surgery/organs/bone_tumor.dm:52]:
```suggestion
```
I think this is all just the same as the normal bone tumor?

**Review by DGamerL — CHANGES_REQUESTED** [review]

File: code/modules/reagents/chemistry/reagents/medicine.dm [hunk 6]
```diff
@@ -1020,7 +1020,10 @@
 	if(ishuman(M) && prob(5))
 		var/mob/living/carbon/human/H = M
 		if(!H.get_int_organ(/obj/item/organ/internal/bone_tumor))
-			new/obj/item/organ/internal/bone_tumor(H)
+			if(isslimeperson(H))
+				new /obj/item/organ/internal/bone_tumor/slime_tumor(H)
+			else
+				new/obj/item/organ/internal/bone_tumor(H)
```
Comment (Henri215) [inline code/modules/reagents/chemistry/reagents/medicine.dm:1026]:
```suggestion
				new /obj/item/organ/internal/bone_tumor(H)
```

**Review by Henri215 — APPROVED** [review]

**Review by DGamerL — APPROVED** [review]

**Pull request merged**